src/dcat_ap_no_validator_service/adapter/remote_graph_adapter.py

```diff
@@ -19,23 +38,26 @@ def __init__(self, message: str) -> None:
         super().__init__(message)
 
 
-def fetch_graph(url: str) -> Graph:
+def fetch_graph(url: str, use_cache: bool = True) -> Graph:
     """Fetch remote graph at url and return as Graph."""
-    logging.debug(f"Trying to fetch remote graph {url}")
+    logging.debug(f"Trying to fetch remote graph {url}.")
     try:
-        resp = requests.get(url, headers={hdrs.ACCEPT: "text/turtle"})
+        if not use_cache:
+            with requests_cache.disabled():
```

Comment:
Gjelder disabling av cachen for kun denne requesten eller alle?

Reply:
Disable gjelder kun for det påfølgande kallet, slik eg har forstått det. Men eg skal sjekke. Godt poeng.
Og sidan cache er ekstern, skal dette vere threadsafe. (Mi naive tolking av dette.)

Reply:
Frå dokumentasjonen til requests-cache:
```
@contextmanager
def disabled():
    """
    Context manager for temporarily disabling caching for all ``requests`` functions

    Example:

        >>> with requests_cache.disabled():
        ...     requests.get('http://httpbin.org/get')

    """
    previous = requests.Session
    uninstall_cache()
    try:
        yield
    finally:
        _patch_session_factory(previous)
```
Ref https://requests-cache.readthedocs.io/en/latest/_modules/requests_cache/patcher.html#disabled

Så disabling gjelder kun for det som er i scope av context (with-clause). Og i vårt tilfelle er den disableing kun for bruker-input.

Reply:
Setter opp automatiske tester som adresser potensielle cache-issues.